### DISCUSSION TITLE
I find the page on JSON Reads/Writes/Format Combinators really, really confusing

### DISCUSSION THREAD

**Discussion by joncard1** [playframework/playframework#11370] [Ideas]
A critical part of the whole piece of performing validation in the method "and", and that's not documented anywhere I can find. I can't even find the source for play.api.libs.functional.syntax. I spent a LOT of time thinking there was a version problem in the documentation and I was supposed to use "andThen."

**Comment by joncard1** [comment]
Just to put a note in here: I'm following the instructions at the bottom of the web page (https://www.playframework.com/documentation/2.8.x/ScalaJsonCombinators), which said to either do a pull request (which I shouldn't do because if my issue with the page is that I don't understand it, I can hardly write the documentation), or go to the discussion page. My inclination would be to file this under Issues, but I'm following instructions.

**Reply by prkarale** [reply]
Wow! nice post, hey check out this cool regenration technique https://rigenerahbw.com

**Reply by octonato** [reply]
This is pure spam. User is blocked from posting here.

**Comment by octonato** [comment]
Thanks for reaching out @joncard1 and apologies for the inconvenient spammer.

Have you tried to ask help in our [Discord chat](https://discord.com/invite/g5s2vtZ4Fa)? Did you manage to do what you wanted?

**Reply by joncard1** [reply]
Sorry for the late reply. I guess I get too much email spam to see this response.

I will ask the Discord chat in the future, thank you. I think I'm good for now.

**Reply by joncard1** [reply]
Oh, and I see the documentation was updated. Thank you!
